File: sample/src/main/java/tech/thdev/androidrecyclerview/view/custom_scroll/header_footer/CustomScrollHeaderFooterActivity.java
package tech.thdev.androidrecyclerview.view.custom_scroll.header_footer;

import android.os.Bundle;
import android.support.annotation.Nullable;

import tech.thdev.androidrecyclerview.R;
import tech.thdev.base.util.ActivityUtilKt;
import tech.thdev.base.view.BaseActivity;

/**
 * Created by Tae-hwan on 01/11/2016.
 */

public class CustomScrollHeaderFooterActivity extends BaseActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_basic_custom_scroll);

        ActivityUtilKt.replaceContentFragment(this, R.id.frame_layout, CustomScrollHeaderFooterFragment.getInstance());
    }
}

File: sample/src/main/java/tech/thdev/androidrecyclerview/ui/header_footer/custom/CustomScrollHeaderFooterActivity.kt
package tech.thdev.androidrecyclerview.ui.header_footer.custom

import android.os.Bundle
import tech.thdev.androidrecyclerview.R
import tech.thdev.base.ui.BaseActivity
import tech.thdev.base.util.replaceContentFragment

/**
 * Created by Tae-hwan on 01/11/2016.
 */
class CustomScrollHeaderFooterActivity : BaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_basic_custom_scroll)

        replaceContentFragment(R.id.frame_layout,
            CustomScrollHeaderFooterFragment.newInstance()
        )
    }
}